rosidl_generator_py/resource/_msg_support.c.em

```diff
@@ -344,14 +344,15 @@ nested_type = '__'.join(type_.namespaced_name())
       char * buffer;
       Py_ssize_t length;
       int rc = PyBytes_AsStringAndSize(encoded_item, &buffer, &length);
-      Py_DECREF(encoded_item);
       if (rc) {
```

Comment:
this is old, but shouldn't be `-1 == rc`?

Reply:
Checking for not equal zero is common for such error code values imo. Therefore I would leave it as is (also since it is orthogonal to this PR).